lib/jsonapi/filtering.rb

```diff
@@ -1,10 +1,4 @@
-begin
-  require 'active_record'
-  require 'ransack'
-  require_relative 'patches'
-rescue LoadError
-  warn('Install `ransack` gem before using `JSONAPI::Filtering`!')
```

Comment:
@gagalago I think we can safely remove this warning, but leave the rest as it is. 

The thing is that active record is needed for ransack, and only for that. So leaving active-record outside of this rescue block, suggests it should be a hard dependency (which is not).

Reply:
meaning that you prefer that I rescue the `LoadError` and hide it by doing nothing? or that I move the `require 'active_record'` near the other require that I already moved?

Reply:
I think it is fine if we just silently ignore the `LoadError`. Those who ignore the post-install message, should know what they are doing :wink: